templates/templates/_navigation-enterprise-h.html

```diff
@@ -16,10 +16,10 @@ <h4 class="p-heading-four is-dense">
             </a>
           </div>
           <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
-            <a href="/openstack">OpenStack solutions&nbsp;&rsaquo;</a>
+            <a href="/openstack">OpenStack&nbsp;&rsaquo;</a>
           </h4>
           <ul class="p-text-list--small is-bordered">
-            <li class="p-list__item u-show--small u-hide--medium u-hide--large"><a href="/openstack">Get OpenStack</a></li>
+            <li class="p-list__item"><a href="/openstack">Get OpenStack</a></li>
```

Comment:
I think we want to keep hiding these on large screens, otherwise you have a "Get OpenStack" CTA followed immediately by a "Get OpenStack" link

Reply:
![screenshot_7](https://user-images.githubusercontent.com/25733845/49213914-4d891c80-f3bd-11e8-8b90-108407409465.png)

Reply:
ah... a greedy search and replace... good catch